tenant_schemas/tests/test_tenants.py

```diff
@@ -1,60 +1,109 @@
-import json
-from StringIO import StringIO
-
 from django.conf import settings
 from django.contrib.auth.models import User
-from django.core.management import call_command
-from django.db import connection
+from django.db import connection, transaction
```

Comment:
Please move to a separate PR if parallel migrations does not cause this test to fail.